invenio_rdm_records/oai.py

```diff
@@ -26,9 +26,9 @@
 from .services.pids.providers.oai import OAIPIDProvider
 
 
-def dublincore_etree(pid, record):
+def dublincore_etree(pid, record, **serializer_kwargs):
```

Comment:
In case the `serializer` value for an OAI metadata format in the `OAISERVER_METADATA_FORMATS` config item happens to be a tuple, the second part of that tuple is being passed as a keyword argument to the function referenced by the first part. See: https://github.com/inveniosoftware/invenio-oaiserver/blob/master/invenio_oaiserver/utils.py#L53-L54